fhir-path/src/main/java/com/ibm/fhir/path/FHIRPathAbstractNode.java

```diff
@@ -28,56 +28,92 @@ protected FHIRPathAbstractNode(Builder builder) {
         children = Collections.unmodifiableCollection(builder.children);
     }
     
+    /**
+     * {@inheritDoc}
+     */
```

Comment:
I think `@inheritDoc` is just if you want to combine some new text with the parent text.  If you're not adding anything, just leave it off.  But maybe double-check me :-)

Reply:
Removed unnecessary `{@inheritDoc}` usage.